src/less/designguide.less

```diff
@@ -176,14 +176,11 @@
 
     #loading-component {
         display: flex;
-        flex-direction: column;
-        align-items: center;
         justify-content: center;
-        height: ~"calc(100vh - 70px)";
+        height: ~"calc(100vh - 156px)";
```

Comment:
Will this break if do anything that will change the height of the footer and header?

Reply:
Yes. But this is just for the DesignGuide. That px value is footer + header + brandbar

Reply:
But why do you need it? If your not going to center the loader?

Reply:
Ah, I don't. I mistook it for the height value used to offset the footer. I'll remove it.